views/campgrounds/edit.ejs

```diff
@@ -1,13 +1,16 @@
 <%- include("../partials/header") %>
 <div class="container">
     <h2>Edit <%= campground.name %> </h2>
-    <form action="/campgrounds/<%= campground._id %>?_method=PUT" method="POST">
+    <form action="/campgrounds/<%= campground._id %>?_method=PUT" method="POST" enctype="multipart/form-data">
```

Comment:
what is that? `enctype="multipart/form-data"`

I have no idea.

Reply:
This allow to send files to the server through a html form